root/statistics/Index.js

```diff
@@ -72,69 +89,69 @@ const Index = ({
           </tr>
           <tr>
             <th>{l('Artists:')}</th>
-            <td colSpan="3">{formatCount($c, stats['count.artist'])}</td>
+            <td colSpan="3">{fc('artist')}</td>
           </tr>
           <tr>
             <th>{l('Release Groups:')}</th>
             <td colSpan="3">
-              {formatCount($c, stats['count.releasegroup'])}
+              {fc('releasegroup')}
```

Comment:
Maybe we should now make it so that all the {fc()} calls are in one line with the tds, unless they don't fit at all? This looks like it'd fit.

Reply:
Yep, that'd make sense. Fixed all the ones I saw.